### PR TITLE
Fix package name resolving for longer URLs

### PR DESCRIPTION
**Pull request by orgads**
Example: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/storage-blob/

### CODE REVIEW

**Comment by BePo65** [comment]
Ok, i see the bug.
Just to be sure that this not only solves a problem for a single project: do you have another similar package to inspect (I looked at angular, which IMHO is a monorepo too, but they have another project structure).

PS: it's a great idea to have tests for extracting the package name. I should have made it in the beginning.

File: test/web-service.spec.js [hunk 1]
```diff
@@ -83,6 +83,49 @@ describe('web-service', () => {
...
+    const testCase = (ctx, expectedName) => {
+      const res = WebService.getPackageNameFromLink(ctx?.title);
+      expect(res).to.equal(expectedName);
+    };
+
+    it('git+https://github.com/foo/bar.git', function() {
+      testCase(this.test, 'foo/bar');
+    })
+
+    it('git+https://github.com/foo/bar.git/', function() {
```
Comment (BePo65) [inline test/web-service.spec.js:96]: Isn't this the same as the first test?

**Comment by orgads** [comment]
https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity/README.md

**Comment by orgads** [comment]
Found another case and handled it: git+https://github.com/grpc/grpc-node.git#master

**Review by BePo65 — APPROVED** [review]
ok, I see the slash

**Comment by orgads** [comment]
Thank you for merging all the PRs. Can you please create a new release? :)

**Comment by BePo65** [comment]
done

**Pull request merged**